usage/install-on-desktops.md

```diff
@@ -54,12 +54,18 @@ sudo xbps-install -S waydroid
 
 ## Ubuntu/Debian and derivatives
 
-These instructions work for ubuntu focal, ubuntu hirsute, debian bullseye, droidian, ubports, and likely more. We will continue to update this document as the project further develops
+These instructions work for Ubuntu focal and hirsute, Debian sid and bullseye, droidian, ubports, and likely more. We will continue to update this document as the project further develops.
 
-### Install Pre-requisites
+### Some Pre-Instructions
+
+1) WayDroid was born to run on Wayland sessions;
+2) WayDroid runs better on Kernel above 5.10.
+
+
+### Install Pre-Requisites
 
 ```bash
-sudo apt install curl ca-certificates -y
+sudo apt install curl ca-certificates lxc gnupg2 apt-transport-https -y
```

Comment:
`lxc` is pulled in as a dependency of waydroid, so is not necessary here.
`gnupg2` is not necessary.
`apt-transport-https` is void: `This is a dummy transitional package - https support has been moved into the apt package in 1.5. It can be safely removed.`